index.html

```diff
@@ -28,25 +28,24 @@ <h3><button>Sign up</button></h3>
 
     <main>
         <section class="box1"></section>
-        
-        <section class="box3"></section>
-        <p>this is some subtext under an illustration or image</p>
         <div class="box2"><img src="./image/IMG-20220320-WA0110.jpg" alt="an animation" width="125px;" height="117px;"></div>
```

Comment:
You have some inline CSS here. Always ensure that all CSS declarations are made in CSS files. Kindly update this tag.

Reply:
I have it there to style the image directly rather than going into CSS files but nevertheless changes have been made.